web/src/main/webapp/xslt/common/base-variables.xsl

```diff
@@ -76,7 +76,7 @@
     else if ($service = 'catalog.viewer') then 'gn_viewer'
     else if ($service = 'catalog.search') then 'gn_search'
     else if ($service = 'md.viewer') then 'gn_formatter_viewer'
-    else 'gn'"/>
+    else 'gn_search'"/>
```

Comment:
it seemed a bug, because if i leave this on gn, i get a css-not-found-error, but i'll have a look